paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/watermark/CdcWatermarkStrategy.java

```diff
@@ -56,13 +56,10 @@ public void onEvent(String record, long timestamp, WatermarkOutput output) {
                     throw new RuntimeException(e);
```

Comment:
I don't think there is a need to throw an exception here. From my understanding, even if there is a problem with the watermark calculation, it will not cause problems with the task, so I think you can just ignore and printing a warning log here. Avoid bugs caused by issues we haven't considered. But this is obviously not the issue that this PR needs to deal with

WDYT @MonsterChenzhuo